Example/SnapshotTests/HighlightTests.swift

```diff
@@ -48,7 +48,7 @@ final class HighlightTests: SnapshotTestCase {
 
 // MARK: -
 
-private final class AccessibleContainerView: UIView {
+final class AccessibleContainerView: UIView {
```

Comment:
Oops, meant to move this view to its own file now that it's shared across test classes

Reply:
Addressed in cc6087d.